test/e2e/far_e2e_test.go

```diff
@@ -455,12 +455,12 @@ func checkRemediation(nodeName string, nodeBootTimeBefore time.Time, oldPodCreat
 	By("Getting new node's boot time")
 	wasNodeRebooted(nodeName, nodeBootTimeBefore)
 
-	By("checking if old VA has been deleted")
-	checkVaDeleted(va)
-
 	By("checking if old pod has been deleted")
 	checkPodDeleted(pod)
 
+	By("checking if old volume attachment has been deleted")
```

Comment:
Why the change in order? AFAIK in a real world the pod won't be deleted before its related VA is deleted (that's why we delete the VA actually). So first checking the VA makes more sense to me.